### ISSUE TITLE
Alt text missing

### ISSUE THREAD

**Issue by JacquelineOmollo** [unitymakesus/justgetsimple#74]
- [x] Founded in 1993
- [x] Bowl a thon
- [x]  Picnic

**Comment by JacquelineOmollo** [comment]
Done
